### ISSUE TITLE
assertj-guava dependence to assertj-core

### ISSUE THREAD

**Issue by amergey** [joel-costigliola/assertj-guava#21]
Latest assertj-guava (1.3.1) depends on assertj-core [1.7.1, 1.8.0), so it seems it is not supposed to be used with more recent version of assertj-core (2.0.0 or 3.0.0).
There should be a assertj-guava version available with assertj-core 2 and 3

**Comment by joel-costigliola** [comment]
agreed

**Comment by joel-costigliola** [comment]
adding an exclude section in assertj-guava dependency solves the problem:
```java
<dependency>
  <groupId>org.assertj</groupId>
  <artifactId>assertj-core</artifactId>
  <version>2.0.0</version>
  <scope>test</scope>
</dependency>
<dependency>
  <groupId>org.assertj</groupId>
  <artifactId>assertj-guava</artifactId>
  <version>1.3.1</version>
  <scope>test</scope>
  <exclusions>
    <exclusion>
      <groupId>org.assertj</groupId>
      <artifactId>assertj-core</artifactId>
    </exclusion>
  </exclusions>
</dependency>
``` 
In that case assertj-core will be resolved to 2.0.0.

Anyway, I should probably release assertj-guava 2.0 that will rely on assertj-core 2.x and  assertj-guava 3.0 that will rely on assertj-core 3.x.
